### DISCUSSION TITLE
Converting vector tiles to raster tiles

### DISCUSSION THREAD

**Discussion by dwoznicki** [go-spatial/tegola#849] [Q&A] [answered]
Hi, thanks for this great project. I have a slightly out-of-scope question.

Do you know of any way to convert the PBF (?) tile files generated by Tegola into rasterized images on the server? My goal would be to do the conversion at request time so that we could serve both raster and vector tiles, and have the styling be consistent.

**Comment by ARolek** [comment, marked as answer]
@dwoznicki not really. Awhile back we were playing with building some [bindings to MapboxGL Native](https://github.com/go-spatial/go-mbgl) but the project never stabilized. It is possible though, it just need someone to run with the rest of the build out and the subsequent maintenance.

**Reply by dwoznicki** [reply]
Thanks for the reply, @ARolek. I'm thinking about spending a bit of time trying to get server side rasterization working with Tegola. This is pretty far from my area of knowledge in many, many ways, so we'll see how it goes.

**Comment by dBitech** [comment]
You could possibly infer some ideas from how this is done in https://github.com/consbio/mbgl-renderer
